### ISSUE TITLE
Default and stable branch - how to set a stable branch as default

### ISSUE THREAD

**Issue by ajithvs** [frej/fast-export#245]
Hi 
Thanks for this excellent tool. 
In our scenario, we are using Mercurial and we have a 'default' branch and a 'stable' branch. This stable branch is equivalent to live and we will release hotfix from this branch. My question is how can I set this stable branch as a master branch and keep default as it is. Our default branch is the branch in which active development is going on. For releases, we will create a release branch from default and once it is released to live we will merge it with the Stable branch. 

Could you please help me to solve this problem?

In short 
Stable - Change tp master 
Default - Keep as it is 

We are going to use Azure DevOps as a git repo.

**Comment by frej** [comment]
Use a branch mapping file -B (see the "Tag and Branch Naming" in the
README) to control what your branches end up as on the git side. If you
cannot or don't want to do it statically, you could also use a plugin to
do it on the fly.

**Comment by frej** [comment]
@ajithvs please close this issue if your question has been answered.
